src/test/java/com/github/redouane59/twitter/unit/TweetStreamConsumerTest.java

```diff
@@ -0,0 +1,52 @@
+package com.github.redouane59.twitter.unit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import com.github.redouane59.twitter.TwitterClient;
+import com.github.redouane59.twitter.dto.tweet.TweetV2;
+import com.github.redouane59.twitter.helpers.TweetStreamConsumer;
+
+import org.junit.jupiter.api.Test;
+
+public class TweetStreamConsumerTest {
+
+
+  @Test
+  void consumeMultipleTweet() throws Exception {
+    // Consumes a Json in 2 passes to simulate two different buffers.
+    TweetStreamConsumer consumer = new TweetStreamConsumer();
+    File                file     = new File(getClass().getClassLoader().getResource("tests/multiple_tweet_stream_example_part1.data").getFile());
+
+    String content = Files.readString(file.toPath());

```

Comment:
This is not working with jdk8 @Greg-M73 , i just realized it now running it.

Reply:
replacing it by `    String content = new String(Files.readAllBytes(Paths.get(file.getPath())));`

Reply:
Yes, just saw it, only available since 11.